### PR TITLE
Fixes 9054 (part 5, com_content Featured). [REGRESSION] sort order

### PR DESCRIPTION
**Pull request by ghost**
See https://github.com/joomla/joomla-cms/issues/9054 for instructions.
Fixes com_content Featured view.

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 0eb1f343cb3aef4fd0e833fd804fb75ae598ca9e<br /><br />Tested with success on current staging. Before patch: Sort order changed from ID Descending to Title Ascending after paging (or changing items per page). After patch: Sort order is Title Ascending and stays like this when paging. In opposite to test description in #9054 there was no need to clear any cookies for this test.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9061">issues.joomla.org/joomla-cms/9061</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 0eb1f343cb3aef4fd0e833fd804fb75ae598ca9e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9061">issues.joomla.org/joomla-cms/9061</a>.</sub>

**Comment by brianteeman** [comment]
RTC - thanks for the quick fix and test<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9061">issues.joomla.org/joomla-cms/9061</a>.</sub>

**Comment by richard67** [comment]
Let's hope it goes into Beta 3 ... maybe someone can set the milestone to 3.5.0 to force that?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9061">issues.joomla.org/joomla-cms/9061</a>.</sub>

File: administrator/components/com_content/models/featured.php [hunk 1]
```diff
@@ -202,4 +202,21 @@ protected function getListQuery($resolveFKs = true)
...
+	 * Method to auto-populate the model state.
+	 *
+	 * Note. Calling getState in this method will result in recursion.
+	 *
+	 * @param   string  $ordering   An optional ordering field.
+	 * @param   string  $direction  An optional direction (asc|desc).
+	 *
+	 * @return  void
+	 *
+	 * @since   1.6
```
Comment (andrepereiradasilva) [inline administrator/components/com_content/models/featured.php]: 3.5?

**Comment by wilsonge** [comment]
Setting the milestone to 3.5 forces nothing FYI. If I disagree with a milestone I won't hesitate to change it - especially at this stage of the release process.

Either way in this case it's fine. Merging<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9061">issues.joomla.org/joomla-cms/9061</a>.</sub>

**Comment by bertmert** [comment]
@andrepereiradasilva 

> * @since 1.6

> * @since 3.5?

It's not easy for "dummies" like me to decide which "since" is correct whenever I have to decide.
Sometimes I see "framework since", sometimes "Joomla CMS since", sometimes without any hint, just a version. Sometimes a funny mixture. Sometimes a wrong "since" when I start investigations for a specific change or feature (and end up frustrated).

function populateState is since 1.6 (is it?) in Featured model it is since 3.5 from now on or is it still 1.6?

Sometimes one doesn't know if or when sth. will be merged. And so on....

Know what I mean ;-)

File: administrator/components/com_content/models/featured.php (hunk 1 above)
Comment (wilsonge) [inline administrator/components/com_content/models/featured.php]: Done with https://github.com/joomla/joomla-cms/commit/8ed60ff340296ac03c727658c69130555811b832

**Pull request merged**